src/js/components/user/login.tsx

```diff
@@ -53,10 +53,11 @@ const Login = ({ user }: { user: User | null }) =>
   user ? (
     <Redirect to={routes.home()} />
   ) : (
-    <div className="slds-welcome-mat slds-welcome-mat_info-only">
-      <div className="slds-welcome-mat__content slds-grid">
+    <div className="slds-welcome-mat slds-welcome-mat_info-only welcome-container">
```

Comment:
Any reason for not using the react [`<WelcomeMat/>` component](https://react.lightningdesignsystem.com/components/welcome-mat/) instead?